CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+## v0.4.0
+Fri Dec 20 16:24:29 PST 2019
+
+* [361ed2d](https://github.com/hyperledger/fabric-chaincode-evm/commit/361ed2d) [FABCE-145](https://jira.hyperledger.org/browse/FABCE-145) update README for 0.4.0 release
+* [bf3fbd5](https://github.com/hyperledger/fabric-chaincode-evm/commit/bf3fbd5) [FAB-16919](https://jira.hyperledger.org/browse/FAB-16919) Remove scripts/goListFiles.sh
+* [0ed01b6](https://github.com/hyperledger/fabric-chaincode-evm/commit/0ed01b6) [FAB-17053](https://jira.hyperledger.org/browse/FAB-17053) Switch Fab3 and Integration to Go Mod
+* [0476cc1](https://github.com/hyperledger/fabric-chaincode-evm/commit/0476cc1) [FAB-17053](https://jira.hyperledger.org/browse/FAB-17053) Switch EVMCC to go modules
+* [3dae2db](https://github.com/hyperledger/fabric-chaincode-evm/commit/3dae2db) [FAB-16918](https://jira.hyperledger.org/browse/FAB-16918) Restructure evmcc directory
```

Comment:
Possibly mention this that the directory structure is slightly different now

Reply:
Not 100% sure what to say about this. Shouldn't impact anyone unless they were depending on semi-internal stuff.

Reply:
Maybe not that specific commit but with the combo of the vendor split, it means chaincode packaging is possible with the evm because of the new directory structure